maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -277,6 +279,26 @@ class Orchestra(
         return true
     }
 
+    private fun scrollUntilVisible(command: ScrollUntilVisibleCommand): Boolean {
+        val endTime = System.currentTimeMillis() + command.timeout
+        var error: MaestroException.ElementNotFound? = null
+        do {
+            try {
+                maestro.waitForAppToSettle()
+                findElement(command.selector, 500)
+                return true
+            } catch (ignored: MaestroException.ElementNotFound) {
+                error = ignored
```

Comment:
Wouldn't this pass the command if the element is not found? Or command would fail?

Reply:
Not sure i understand. If the element is not found within the timeout, an exception is thrown

Reply:
Yup so I was confused would it be Timeout or Element not found?

Reply:
Got the answer below 👍